### PR TITLE
IDE-4044 gradle build script sync should be enabled by default in studio

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by jtydhr88** [comment]
jenkins retest

File: tools/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/LiferayUIPlugin.java [hunk 1]
```diff
@@ -339,6 +341,28 @@ public void run() {
...
 
+	private void _enableGradleAutoSync() {
+		IEclipsePreferences gradlePrefs = InstanceScope.INSTANCE.getNode("org.eclipse.buildship.core");
+
+		String autoSync = "auto.sync";
+
+		try {
+			String[] gradlePrefsKeys = gradlePrefs.keys();
+
+			Stream<String> gradlePrefsKeysSteam = Stream.of(gradlePrefsKeys);
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/LiferayUIPlugin.java]: This should be done in the product_customization.ini file here:  https://github.com/liferay/liferay-ide/blob/master/enterprise/plugins/com.liferay.ide.studio.ui/product_customization.ini

**Comment by jtydhr88** [comment]
hey @Camork, can you make the change and resend, thanks.

File: tools/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/LiferayUIPlugin.java (hunk 1 above)
Comment (Camork) [reply tools/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/LiferayUIPlugin.java]: Hey ,greg. 
    I tried to use what you said  to implement, but it couldn't be done. The reason:
buildship not use `UIPlugin.getInstance().getPreferenceStore()` to receive client's inputs.
then i tried to change buildship source code to verify it. Is my explanation correct?  You can reference this class [GradleWorkbenchPreferencePage ](https://github.com/eclipse/buildship/blob/master/org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/preferences/GradleWorkbenchPreferencePage.java#L75).

**Comment by gamerson** [comment]
Understand.  Thanks for explaination.

**Comment by gamerson** [comment]
Merged in 0d48c9d750a1dd26ecd8326ff2621ad2cd7ba5af. Thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**